src/basicOp.js

```diff
@@ -24,7 +24,13 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  switch (operation) {
+    case ('+') : return value1 + value2;
```

Comment:
скобочки и пробел перед двоеточием не нужны
необходимо делать переносы
```suggestion
    case '+':
       return value1 + value2;
```